jnosql-mapping/jnosql-mapping-core/src/main/java/org/eclipse/jnosql/mapping/reflection/Reflections.java

```diff
@@ -148,10 +146,17 @@ void makeAccessible(Field field) {
      * @throws ConstructorException when the constructor has public and default
      */
     public <T> Constructor<T> getConstructor(Class<T> type) {
+
+        final Predicate<String> isIdAnnotation = Id.class.getName()::equals;
```

Comment:
How about moving those to static?

Reply:
Yeah... making these predicates static ones will avoid creating Predicate instances ever this method is executed...